### PR TITLE
Simple docker-compose

### PR DESCRIPTION
**Pull request by hguerrero**
Update the docker-compose setup:

- change to docker-compose.yaml
- remove the https config
- mount realm from volume 
- use generic name for host hostname

### CODE REVIEW

**Comment by lbroudoux** [comment]
Hi @hguerrero 

And thanks for PR ! I've tested everything on my Mac and its way easier than I was previously achieved ... I'd like to make a test on a Linux box to be sure that everything is also ok and I'll merge it.

File: install/docker-compose/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+version: '2'
+
+services:
+  mongo:
+    image: mongo:3.4.23
+    container_name: microcks-db
```
Comment (lbroudoux) [inline install/docker-compose/docker-compose.yml:6]: Could you change to the old name that is `microcks-mongo`? That way we know it's the container attached to the `mongo` alias.

File: install/docker-compose/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    volumes:
+      - "~/tmp/microcks-data:/data/db"
+
+  keycloak:
+    image: jboss/keycloak:10.0.1
+    container_name: microcks-sso
```
Comment (lbroudoux) [inline install/docker-compose/docker-compose.yml:12]: Could you change to the old name that is `microcks-keycloak`? That way we know it's the container attached to the `keycloak` alias.

**Review by lbroudoux — CHANGES_REQUESTED** [review]
Just 2 changes regarding container names.

File: install/docker-compose/docker-compose.yml (hunk 1 above)
Comment (hguerrero) [reply install/docker-compose/docker-compose.yml:6]: This name is shown only when interacting with docker like in `docker ps` the alias used in the network level is still `mongo`. Is there a specific need to expose the implementation detail on db and sso ?

**Comment by hguerrero** [comment]
@lbroudoux I think in the case of Linux box you will need to configure the host IP address instead to make it work as I don't recall there is a entry created in the `/etc/hosts` file.

**Comment by hguerrero** [comment]
@lbroudoux this PR now has the updated config required to address #119 and set up the correct external Keycloak URL and frontend hostname.

**Comment by lbroudoux** [comment]
Yes just saw it. Perfect !

File: install/docker-compose/docker-compose.yml (hunk 1 above)
Comment (lbroudoux) [reply install/docker-compose/docker-compose.yml:6]: I use to have many mongo containers running or in the `docker ps` history and love to see that one is coming from microcks. But I'll change it afterwards. No problem ;-)

**Review by lbroudoux — APPROVED** [review]

**Comment by lbroudoux** [comment]
Thanks a lot @hguerrero !!

**Pull request merged**